src/canvas/dialogs/dd_dialog.rs

```diff
@@ -58,6 +66,214 @@ impl KillDialog for Painter {
+
+        f.render_widget(
+            Paragraph::new(yes_button)
+                .block(Block::default())
+                .alignment(Alignment::Right),
+            button_layout[0],
+        );
+        f.render_widget(
+            Paragraph::new(no_button)
+                .block(Block::default())
+                .alignment(Alignment::Left),
+            button_layout[2],
+        );
+
+        if app_state.should_get_widget_bounds() {
+            app_state.delete_dialog_state.button_positions = vec![
+                (
+                    button_layout[2].x,
+                    button_layout[2].y,
+                    button_layout[2].x + button_layout[2].width,
+                    button_layout[2].y + button_layout[2].height,
+                    0,
+                ),
+                (
+                    button_layout[0].x,
+                    button_layout[0].y,
+                    button_layout[0].x + button_layout[0].width,
+                    button_layout[0].y + button_layout[0].height,
+                    1,
+                ),
+            ];
+        }
+    }
+
+    #[cfg(target_family = "unix")]
+    fn draw_dd_confirm_buttons<B: Backend>(
+        &self, f: &mut Frame<'_, B>, button_draw_loc: &Rect, app_state: &mut App,
+    ) {
+        let signal_text = vec![
+            "0: Cancel",
+            "1: HUP",
+            "2: INT",
+            "3: QUIT",
+            "4: ILL",
+            "5: TRAP",
+            "6: ABRT",
+            "7: BUS",
+            "8: FPE",
+            "9: KILL",
+            "10: USR1",
+            "11: SEGV",
+            "12: USR2",
+            "13: PIPE",
+            "14: ALRM",
+            "15: TERM",
+            "16: STKFLT",
+            "17: CHLD",
+            "18: CONT",
+            "19: STOP",
+            "20: TSTP",
+            "21: TTIN",
+            "22: TTOU",
+            "23: URG",
+            "24: XCPU",
+            "25: XFSZ",
+            "26: VTALRM",
+            "27: PROF",
+            "28: WINCH",
+            "29: IO",
+            "30: PWR",
+            "31: SYS",
+            "34: RTMIN",
+            "35: RTMIN+1",
+            "36: RTMIN+2",
+            "37: RTMIN+3",
+            "38: RTMIN+4",
+            "39: RTMIN+5",
+            "40: RTMIN+6",
+            "41: RTMIN+7",
+            "42: RTMIN+8",
+            "43: RTMIN+9",
+            "44: RTMIN+10",
+            "45: RTMIN+11",
+            "46: RTMIN+12",
+            "47: RTMIN+13",
+            "48: RTMIN+14",
+            "49: RTMIN+15",
+            "50: RTMAX-14",
+            "51: RTMAX-13",
+            "52: RTMAX-12",
+            "53: RTMAX-11",
+            "54: RTMAX-10",
+            "55: RTMAX-9",
+            "56: RTMAX-8",
+            "57: RTMAX-7",
+            "58: RTMAX-6",
+            "59: RTMAX-5",
+            "60: RTMAX-4",
+            "61: RTMAX-3",
+            "62: RTMAX-2",
+            "63: RTMAX-1",
+            "64: RTMAX",
```

Comment:
Does this list of signals hold for macOS?  I tried using 64 and it didn't work, for example.